lnwallet/chainfee/estimator_test.go

```diff
@@ -107,17 +107,20 @@ func TestSparseConfFeeSource(t *testing.T) {
 		2: 42,
 		3: 54321,
 	}
-	testJSON := map[string]map[uint32]uint32{
-		"fee_by_block_target": testFees,
+	testMinRelayFee := SatPerKVByte(1000)
```

Comment:
let's add a backwards compatibility test here using the old struct? 

```
type jsonResp struct {	
		FeeByBlockTarget map[uint32]uint32 `json:"fee_by_block_target"`	
}
```

Reply:
Added a new test.